MSALiOSB2C/ViewController.swift

```diff
@@ -184,35 +153,13 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
                     }
             }
         } catch {
-            self.loggingText.text = "Unable to create application \(error)"
+            self.loggingText.text = "Unable to create application: \(error)"
         }
     }
 
     @IBAction func refreshToken(_ sender: UIButton) {
-        
-        /**
-         The way B2C knows what actions to perform for the user of the app is through the use of `Authority URL`,
-         a URL indicating a directory that MSAL can use to obtain tokens. In Azure B2C
-         it is of the form `https://<instance/tfp/<tenant>/<policy>`, where `<instance>` is the
-         directory host (e.g. https://login.microsoftonline.com), `<tenant>` is a
-         identifier within the directory itself (e.g. a domain associated to the
-         tenant, such as contoso.onmicrosoft.com), and `<policy>` is the policy you wish to
-         use for the current user flow. The policy we are using here is the `kSignupOrSigninPolicy`
-         *as that is the policy we originally acquired the token with*. It is very important that all actions
-         against a token acquired by a policy maintains the user of that policy on each call.
-         
-         Note the fact that we also look for `InteractionRequired` as an error code and
-         prompt the user interactively. Often times the inability to use a refresh token
-         is from either a password change, refersh token expiring, or other event that
-         can be remedied by the user signing in again. This shouldn't be necessary at every
-         `AcquireTokenSilent()`. If you are experiencing that in your application, make
-         sure you are using the cache correctly and using the same authority.
-         */
-        
-        let kAuthority = String(format: kEndpoint, kTenantName, kSignupOrSigninPolicy)
-        
         do {
-            
+            let kAuthority = try self.getAuthority(forPolicy: self.kSignupOrSigninPolicy)
```

Comment:
nit:in future, let's start using guards more. Even in sample, it would allow us to have more granular error handling.